### ISSUE TITLE
Migrations fail when upgrading to 0.16.1 from 0.15.2

### ISSUE THREAD

**Issue by igorcoding** [infrahq/infra#3607]
## Describe the bug

Upgrading to 0.16.1 from 0.15.2 fails running migration that creates a trigger (logs below). UI component runs ok, but server fails on a migration and enters CrashloopBackoff state

## Environment

```
$ kubectl version
```

Client Version: version.Info{Major:"1", Minor:"25", GitVersion:"v1.25.3", GitCommit:"434bfd82814af038ad94d62ebe59b133fcb50506", GitTreeState:"clean", BuildDate:"2022-10-12T10:47:25Z", GoVersion:"go1.19.2", Compiler:"gc", Platform:"darwin/amd64"}
Kustomize Version: v4.5.7
Server Version: version.Info{Major:"1", Minor:"20", GitVersion:"v1.20.15", GitCommit:"8f1e5bf0b9729a899b8df86249b56e2c74aebc55", GitTreeState:"clean", BuildDate:"2022-01-19T17:23:01Z", GoVersion:"go1.15.15", Compiler:"gc", Platform:"linux/amd64"}

## Relevant Infra Logs

```
{"level":"info","time":1668093401359,"caller":"migrator/migrator.go:197","message":"Running migration 2022-09-01T15:00"}
{"level":"warn","rows":0,"query":"\nALTER TABLE identities\n\tADD COLUMN IF NOT EXISTS verified boolean NOT NULL DEFAULT false,\n\tADD COLUMN IF NOT E
XISTS verification_token text NOT NULL DEFAULT substr(replace(translate(encode(decode(MD5(random()::text), 'hex'),'base64'),'/+','=='),'=',''), 1, 10)
;\n\nCREATE UNIQUE INDEX IF NOT EXISTS idx_identities_verified ON identities (organization_id, verification_token) WHERE (deleted_at IS NULL);","elaps
ed":1058.832869,"time":1668093402418,"caller":"data/data.go:165"}
{"level":"info","time":1668093402419,"caller":"migrator/migrator.go:197","message":"Running migration 2022-09-22T11:00"}
{"level":"info","time":1668093402453,"caller":"migrator/migrator.go:197","message":"Running migration 2022-09-22T13:00:00"}
{"level":"info","time":1668093402457,"caller":"migrator/migrator.go:197","message":"Running migration 2022-10-04T11:44"}
{"level":"info","time":1668093402459,"caller":"migrator/migrator.go:197","message":"Running migration 2022-10-05T11:12"}
{"level":"info","time":1668093402462,"caller":"migrator/migrator.go:197","message":"Running migration 2022-10-05T18:00:00"}
{"level":"info","time":1668093402464,"caller":"migrator/migrator.go:197","message":"Running migration 2022-09-28T13:00"}
{"level":"info","time":1668093402468,"caller":"migrator/migrator.go:197","message":"Running migration 2022-10-17T18:00"}
{"level":"error","rows":0,"query":"\nCREATE OR REPLACE FUNCTION grants_notify() RETURNS trigger\n\tLANGUAGE PLPGSQL\n\tAS $$\nBEGIN\nPERFORM pg_notify
(current_schema() || '.grants_' || NEW.organization_id, row_to_json(NEW)::text);\nRETURN NULL;\nEND; $$;\n\nCREATE OR REPLACE TRIGGER grants_notify_tr
igger AFTER insert OR update\nON grants\nFOR EACH ROW EXECUTE FUNCTION grants_notify();\n","elapsed":11.010055,"error":"ERROR: syntax error at or near
 \"TRIGGER\" (SQLSTATE 42601)","time":1668093402481,"caller":"data/data.go:165"}
Error: creating server: db: migration failed: failed to apply migration 2022-10-17T18:00: ERROR: syntax error at or near "TRIGGER" (SQLSTATE 42601)
```
## Additional context

We use a managed PostgreSQL database in https://selectel.ru/

**Comment by igorcoding** [comment]
Is PostgreSQL 12 not supported by Infra?

**Comment by dnephin** [comment]
Thank you for the bug report! I believe this was fixed in #3602. We are now testing against postgres 12, so it should be supported going forward.

**Comment by igorcoding** [comment]
Okay, thank you! Looking forward to it

**Comment by igorcoding** [comment]
Hi! I've noticed you published a 0.17.0 version which apparently has a mentioned PR merged - this is awesome! Could you please also publish a new helm chart with the 0.17.0 release?

**Comment by mchiang0610** [comment]
@igorcoding new release + helm chart is out for 0.17.1 

This should resolve your issue. I'm going to close it. If you still run into issues, please feel free to reopen this.
